.codecov.yml

```diff
@@ -1,2 +1,7 @@
 ignore:
-  - "example/*"
+  - "cmake-build-debug/**"
+  - "cmake-modules/**"
+  - "doc/**"
+  - "external/**"
+  - "test/**"
```

Comment:
Why you ignoring these?

Reply:
Copied this from the BoostReal repo, not sure what exclusion means in that context but thought it meant that we are not getting the coverage result of those, keeping only the coverage of the library itself.